dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -29,6 +29,19 @@ class Census::StateCsOffering < ApplicationRecord
     SC
   ).freeze
 
+  # By default we treat the lack of state data for high schools as an
+  # indication that the school doesn't teach cs. We aren't as confident
+  # that the state data is conplete for the following states so we do
+  # no want to treat the lack of data as a no for those.
```

Comment:
nit: not.  That said, useful and actionable comment, which is great.

Reply:
I wonder how many of my commits are 'Fix typo'?

Reply:
I checked: 15 typo commits out of 312 (4.8%)
Seems kind of high :(